### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by DatGuyJonathan** [jaredLunde/masonic#14]
I wanted to say thanks for authoring this module. I'm not a new developer, but I'm new to ReactJS and I was playing around with masonry. I first tried `react-virtualized`, and although it seems powerful with lots of controls, I couldn't get an infinite masonry working. Maybe I didn't read the docs hard enough. But, I was able to get off the ground pretty quickly with this package. Also, +1 for doing this in TypeScript. My background is more class-based object-oriented languages so the types definitely helped.

Maintaining an open-source project is a thankless job, so thank you so much!

**Comment by jaredLunde** [comment]
Wow, thank you for the kind words. I got chills - it means a lot to hear this kind of thing. Let me know if you run into any issues or have any questions! Happy to help.

**Comment by BradRyan** [comment]
YES. I second this.  I probably spent the better part of 2 hours trying to get react virtualized trying to do what I want, and the resulting UX was still pretty janky.  Stumbled upon your package and was able to get things working as preferred (think you picked some great reasonable defaults) in no time. And so little code.

Thanks for publishing ❤️

**Comment by gadicc** [comment]
And... 2.5 yrs later I felt compelled to add some additional THANKS for this package which is just pure awesomeness.   I also struggled with react-virtualised masonry layout for a few hours, then found this... it's honestly like magic, and my site feels like it's on steroids now.  Thank you so so so much for all your incredible work on this which is such an asset to the rest of us! :pray: :pray: :pray: 

I guess you don't want to leave stray open issues but I feel like this one might be nice to leave open as I'm sure others would love an obvious place to express their gratitude too... but do understand wanting minimize open issues :sweat_smile:

**Comment by maierson** [comment]
Amazing library. I fully agree with the thoughts and thanks. Thank you Jared.

**Comment by olliejm** [comment]
Thought I'd chime in to also say thanks here. I needed to create a windowed, infinitely scrolling and responsive grid-list, and was surprised to find no easy premade solutions for what I would have thought is a common UI pattern - that is, until finding this library! 

Seemed like I was gonna have to go with react-virtualized otherwise, and it would've probably eaten up a good couple hours of time figuring it out, but I was able to get the full functionality I needed with this in about 2 minutes, and a neat little 20 line component. The performance is excellent as well. Thank you!
